inbm/cloudadapter-agent/fpm-template/usr/bin/provision-tc

```diff
@@ -85,25 +85,17 @@ function check_requirements {
     return 0
   fi
 
+  verified_os_list=("Ubuntu 18.04" "Ubuntu 20.04" "Ubuntu 21.10")
 
-  if [ "$(lsb_release -rs)" == "18.04" ]; then
-    echo "Confirmed Supported Platform (Ubuntu 18.04)"
-  elif [ "$(lsb_release -rs)" == "20.04" ]; then
-    echo "Confirmed Supported Platform (Ubuntu 20.04)"
-  elif [ "$(lsb_release -rs)" == "21.10" ]; then
-    echo "Confirmed Supported Platform (Ubuntu 21.10)"
+  if [[ ${verified_os_list[@]} == *"$(lsb_release -rs)"* ]]; then
+    echo "Confirmed Supported Platform (Ubuntu $(lsb_release -rs))"
```

Comment:
If Debian 10, "Buster", is not tested in our integration tests then we should remove the elif condition as well. Since we are allowing installation to proceed.

Same comment of the other instances as well.